index.html

```diff
@@ -70,5 +70,314 @@ <h1 class="headline_title">
         </ul>
       </div>
     </section>
-  </body>
-</html>
+    <section>
+        <div>
+         <div>
+             <h1 class="batu">
+                 My Recent <br> Works
+             </h1><hr>
```

Comment:
- [ ] Kindly implement the [required changes](https://github.com/JansueT/Portfolio/pull/2/files#r1038804753) from the previous reviewer. 

![Screenshot from 2022-12-04 01-12-26](https://user-images.githubusercontent.com/60083437/205457869-a8de57b7-d026-4cf3-a367-5659b7252528.png)